Testing/Unit/PowerShell/Orchestrator/Invoke-ScubaConfig.Tests.ps1

```diff
@@ -0,0 +1,74 @@
+$OrchestratorPath = '../../../../PowerShell/ScubaGear/Modules/Orchestrator.psm1'
+$ScubaConfigPath = '../../../../PowerShell/ScubaGear/Modules/ScubaConfig/ScubaConfig.psm1'
+$TestUtilsPath = "../PsTestUtils.psm1"
+Import-Module (Join-Path -Path $PSScriptRoot -ChildPath $OrchestratorPath) -Function 'Invoke-SCuBA' -Force
+Import-Module (Join-Path -Path $PSScriptRoot -ChildPath $ScubaConfigPath)
+Import-Module (Join-Path -Path $PSScriptRoot -ChildPath $TestUtilsPath)
+
+InModuleScope Orchestrator {
+    Describe -Tag 'Orchestrator' -Name 'Invoke-Scuba with Config' {
+        BeforeAll {
+            function Remove-Resources {}
```

Comment:
Do not define a new function.  We want to mock the existing function, Remove-Resources.  This applies to all of these mocked functions.

Reply:
Not clear on use of mocked functions  For example the declaration of InvokeScubaTests.ps1:Invoke-ReportCreation seems inconsistent with the declaration  surrounding it
```
function Invoke-RunRego {}
Mock -ModuleName Orchestrator Invoke-RunRego {}

Mock -ModuleName Orchestrator Invoke-ReportCreation {}
 
function Disconnect-SCuBATenant {}
Mock -ModuleName Orchestrator Disconnect-SCuBATenant {}
```

Reply:
Yes, others were not clear on use of mock when initially writing unit tests.  As I see these issues I am pointing them out.

Reply:
So is that above code correct?

Reply:
Not sure exactly what the function declaration does in the above example